.github/workflows/build_flow_visualisation.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
       - name: Build visualisation
         run: |
-          dot -v
+          dot -V
```

Comment:
I'm not sure what this was trying to achieve with `dot -v` - this just hangs (-v is for verbose, -V is for version).
`dot -V` just prints out the version, so useful to see if graphviz is installed.